### PR TITLE
ADS: Create new Dialog Components

### PR DESCRIPTION
**Pull request by malmstein**
Task/Issue URL:  https://app.asana.com/0/1174433894299346/1202861107912491

### Description
This PR creates three new dialogs that will replace all dialogs in the app.
The next PR will swap the old MaterialAlertDialog with the new Components

### CODE REVIEW

File: common-ui/src/main/res/layout/dialog_single_choice_alert.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright (c) 2021 DuckDuckGo
```
Comment (joshliebe) [inline common-ui/src/main/res/layout/dialog_single_choice_alert.xml:4]: *2022

Comment (joshliebe) [inline common-ui/src/main/res/layout/dialog_single_choice_alert.xml:4]:
```suggestion
  ~ Copyright (c) 2022 DuckDuckGo
```

File: common-ui/src/main/res/layout/dialog_stacked_alert.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright (c) 2021 DuckDuckGo
```
Comment (joshliebe) [inline common-ui/src/main/res/layout/dialog_stacked_alert.xml:4]:
```suggestion
  ~ Copyright (c) 2022 DuckDuckGo
```

File: common-ui/src/main/res/layout/dialog_text_alert.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright (c) 2021 DuckDuckGo
```
Comment (joshliebe) [inline common-ui/src/main/res/layout/dialog_text_alert.xml:4]:
```suggestion
  ~ Copyright (c) 2022 DuckDuckGo
```

File: common-ui/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -42,6 +42,11 @@
     <string name="radio_button_label_title">Radio button</string>
     <string name="checkbox_label_title">Checkbox</string>
     <string name="snackbar_label_title">Snackbar</string>
+    <string name="text_dialog_title">Title</string>
+    <string name="text_dialog_message">You can put as much text in here as you like. This dialog scales automagically.</string>
```
Comment (joshliebe) [inline common-ui/src/main/res/values/strings.xml:46]: 😁

**Review by joshliebe — APPROVED** [review]
Just the dates wrong. Can't wait for the new dialogs, LGTM ✅

**Comment by malmstein** [comment]
Thanks @joshliebe!

**Pull request merged**